javascripts/src/mobile-nav.js

```diff
@@ -14,6 +14,16 @@
     this.setup();
   }
 
+  function toggleItem($toggler, $parent, $icon) {
+    $parent.toggleClass('is-open');
+    $parent.toggleClass('is-closed');
+
+    var hidden = ($toggler.siblings('ul').attr('aria-hidden') == 'true') ? 'false' : 'true';
+    $toggler.siblings('ul').attr('aria-hidden', hidden);
+
+    $icon.toggleClass('fa-angle-down fa-angle-up');
+  }
+
   MobileMenu.prototype.setup = function() {
```

Comment:
Since you're calling `toggleItem` from inside another function's scope, you have to save a reference to the class's scope here: 

`var self = this`